compiler/test/compilable/test23479.h

```diff
@@ -0,0 +1,2 @@
+
+int foo() { }
```

Comment:
This is a useless file currently, the test suite is not going to run this

Reply:
It just needs to compile the .h file.

Reply:
But it doesn't, since you didn't change the test runner to also look for .h files in `test/compilable`

Reply:
https://github.com/dlang/dmd/pull/14640